docs/source/howto/queryable-encryption.rst

```diff
@@ -0,0 +1,176 @@
+================================
+Configuring Queryable Encryption
+================================
+
+.. versionadded:: 5.2.0b2
+
+.. admonition:: Queryable Encryption Compatibility
+
+   You can use Queryable Encryption on a MongoDB 7.0 or later replica
+   set or sharded cluster, but not a standalone instance.
+   :ref:`This table <manual:qe-compatibility-reference>` shows which MongoDB
+   server products support which Queryable Encryption mechanisms.
+
+.. _server-side-queryable-encryption:
+
+Configuring Queryable Encryption in Django is similar to
+:doc:`manual:core/queryable-encryption/quick-start` but with some additional
+steps required for Django.
+
+Prerequisites
+-------------
+
+In addition to :doc:`installing </intro/install>` and :doc:`configuring
+</intro/configure>` Django MongoDB Backend, you will need to install some
+additional packages to use Queryable Encryption. This can be done with the
+optional dependency ``encryption`` in the ``django-mongodb-backend`` package.
+
+    pip install django-mongodb-backend[encryption]
+
+.. _server-side-queryable-encryption-settings:
+
+Server-side Queryable Encryption
```

Comment:
_Definitely_ not a phrase we should be using in documentation; the core point of CSFLE/QE is that the encryption itself is primarily a client-side feature, and this title sounds like some of the encryption logic actually happens on the server side. This should be something like "Server-side schema for Queryable Encryption".

Reply:
😂 OK will fix

Reply:
fixed

Reply:
Right, I meant to say "server-side schema configuration" or something like that. I've since removed specific mention of server or client schema configuration, but today I came across this: https://www.mongodb.com/docs/manual/core/queryable-encryption/fundamentals/encrypt-and-query/#client-and-server-schema-validation

Is it accurate to say that in our case, clients without `encrypted_fields_map` are "downloading the schema from the server" ?